docs/excel/configure-your-add-in-to-use-a-shared-runtime.md

```diff
@@ -107,6 +107,9 @@ When you add the `Runtime` element, you also specify a lifetime with a value of
 </Runtimes>
```

Comment:
It looks like the note above this sample is not formatted correctly. Can you take a look and fix? It should be `[!NOTE]` instead of `![NOTE]`.